server/src/test/java/org/elasticsearch/cluster/routing/allocation/allocator/DesiredBalanceComputerTests.java

```diff
@@ -1428,7 +1430,7 @@ private void assertLoggerExpectationsFor(Runnable action, MockLog.LoggingExpecta
 
     public void testLoggingOfComputeCallsAndIterationsSinceConvergence() {
         final var clusterSettings = new ClusterSettings(
-            Settings.builder().put(DesiredBalanceComputer.PROGRESS_LOG_INTERVAL_SETTING.getKey(), TimeValue.timeValueMillis(5L)).build(),
+            Settings.builder().put(DesiredBalanceComputer.PROGRESS_LOG_INTERVAL_SETTING.getKey(), TimeValue.timeValueMillis(1L)).build(),
```

Comment:
I don't think we should do this - at `5ms` some time can pass without seeing another log message (and that's what we want) but at `1ms` we expect a log message every tick of the clock.